docs/dictionary/keyword/else-if.lcdoc

```diff
@@ -0,0 +1,35 @@
+Name: else if
+
+Type: keyword
+
+Syntax: else if
+
+Summary:
+Used within an <if> <control structure> to evaluate an alternate <conditional expression|condition> and contain
+a <statement|statement> or <statementlist|Statement List> to be executed if it's
```

Comment:
`<A|A>` is equivalent to `<A>` in the docs markup, i.e. this can just be `<statement>` (as long as it has a reference below - see comment on references)

Reply:
Also `statement list` doesn't need capitalisation here

Reply:
Should be `its` not `it's`